### PR TITLE
#689 SVarC Model Reorganisation

### PR DESCRIPTION
**Pull request by RomainLosseauRTE**
One model SVarCStandard that includes an BG Injector connected to the B controller

### CODE REVIEW

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Sources/InjectorBG.mo [hunk 1]
```diff
@@ -53,5 +59,7 @@ model InjectorBG "Injector controlled by a the susceptance B and the conductance
     PInjPu = -ComplexMath.real(terminal.V * ComplexMath.conj(terminal.i));
     QInjPu = -ComplexMath.imag(terminal.V * ComplexMath.conj(terminal.i));
 
-annotation(preferredView = "text");
+annotation(preferredView = "diagram",
```
Comment (MarianneSaugierRTE) [inline dynawo/sources/Models/Modelica/Dynawo/Electrical/Sources/InjectorBG.mo]: Thanks for the annotations, the representation of both injectors looks very nice when included in a model such as in SVarCStandard! But the graphical view you added doesn't corresponds to the view "diagram" so I would prefer to keep the preferredView as text (there is currently nothing in the diagram view...)

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Sources/InjectorIDQ.mo [hunk 2]
```diff
@@ -76,7 +75,9 @@ equation
   QInjPu = -1 * ComplexMath.imag(terminal.V * ComplexMath.conj(terminal.i))*SystemBase.SnRef/SNom; //
   PInjPu = -1 * ComplexMath.real(terminal.V * ComplexMath.conj(terminal.i))*SystemBase.SnRef/SNom; //
 
-annotation(preferredView = "text",
-Documentation(info="<html> <p> This block calculates the current references for terminal connection based on d-q-frame setpoints from generator control  </p> </html>"),Icon(graphics = {Rectangle(extent = {{-100, 100}, {100, -100}}), Text(origin = {43, -12}, extent = {{-83, 72}, {-9, -44}}, textString = "Inj")}));
+annotation(preferredView = "diagram",
```
Comment (MarianneSaugierRTE) [inline dynawo/sources/Models/Modelica/Dynawo/Electrical/Sources/InjectorIDQ.mo]: same

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/StaticVarCompensators/package.order [hunk 3]
```diff
@@ -0,0 +1,2 @@
+SVarCStandard
```
Comment (MarianneSaugierRTE) [inline dynawo/sources/Models/Modelica/Dynawo/Electrical/StaticVarCompensators/package.order]: add BaseControls in the first place

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/StaticVarCompensators/SVarCStandard_INIT.mo [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+  Types.ComplexVoltagePu u0Pu "Start value of complex voltage at injector terminal in p.u (base UNom)";
+  Types.ComplexApparentPowerPu s0Pu "Start value of complex apparent power in p.u (base SnRef) (receptor convention)";
+  flow Types.ComplexCurrentPu i0Pu "Start value of complex current at load terminal in p.u (base UNom, SnRef) (receptor convention)";
+
+  Types.PerUnit G0Pu "Start value of the conductance in p.u (base SNom)";
+  Types.PerUnit B0Pu "Start value of the susceptance in p.u (base SNom)";
+  Types.VoltageModule URef0  "Start value of voltage reference in kV";
+
+  InjectorBG_INIT injector(SNom = SNom, U0Pu = U0Pu, UPhase0 = UPhase0, P0Pu = P0Pu, Q0Pu = Q0Pu);
```
Comment (MarianneSaugierRTE) [inline dynawo/sources/Models/Modelica/Dynawo/Electrical/StaticVarCompensators/SVarCStandard_INIT.mo]: This is the first time that we implement such a thing in an init model. Is there a specific reason behind this choice? Do you need this to mimic the behavior that we had with the initConnect in the preassembled model?

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Sources/InjectorBG.mo (hunk 1 above)
Comment (RomainLosseauRTE) [reply dynawo/sources/Models/Modelica/Dynawo/Electrical/Sources/InjectorBG.mo]: Agreed

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/StaticVarCompensators/SVarCStandard_INIT.mo (hunk 4 above)
Comment (RomainLosseauRTE) [reply dynawo/sources/Models/Modelica/Dynawo/Electrical/StaticVarCompensators/SVarCStandard_INIT.mo]: If we want to use the InjectorBG_INIT model, I think we do. We had the same issue with the old HVDC standard model and we ended up with this way of propagating the init parameters.

Comment (MarianneSaugierRTE) [reply dynawo/sources/Models/Modelica/Dynawo/Electrical/StaticVarCompensators/SVarCStandard_INIT.mo]: ok!

**Pull request merged**